dsl/src/test/scala/com/crobox/clickhouse/dsl/JoinQueryIT.scala

```diff
@@ -53,7 +54,7 @@ class JoinQueryIT
   ) { (joinType, result) =>
     ignore should s"join correctly on: $joinType" in {
       val query: OperationalQuery =
-        select(itemId).from(select(itemId).from(TwoTestTable).join(joinType, ThreeTestTable).using(itemId))
+        select(itemId).from(select(itemId).from(TwoTestTable).join(joinType, ThreeTestTable, Option("TTT")).using(itemId))
```

Comment:
This test is ignored?

Reply:
ASOF is a tricky test since it is higly clickhouse specific. I'll look into it

Reply:
Fixed it. Also added assume in order for tests to succeed on clickhouse 19 versions